### PR TITLE
[release/v1.2] release note for v1.2.3

### PR DESCRIPTION
**Pull request by zhaohuabing**

### CODE REVIEW

File: site/layouts/shortcodes/helm-version.html [hunk 1]
```diff
@@ -6,8 +6,8 @@
 {{- "v1.1.4" -}}
 {{- end -}}
 {{- with (strings.HasPrefix $pagePrefix "v1.2") -}}
-{{- "v1.2.2" -}}
+{{- "v1.2.3" -}}
```
Comment (arkodg) [inline site/layouts/shortcodes/helm-version.html:9]: Should this be part of another PR, else this will cause the.docs.site to be broken for some duration

Comment (zhaohuabing) [reply site/layouts/shortcodes/helm-version.html:9]: A better approach could be releasing first, then modify the site on the main branch.

Comment (zhaohuabing) [reply site/layouts/shortcodes/helm-version.html:9]: I think a better approach would be publishing the release first, then modify the doc?

Comment (arkodg) [reply site/layouts/shortcodes/helm-version.html]: Yeah +1

**Review by arkodg — APPROVED** [review]

**Review by zirain — APPROVED** [review]
